### ISSUE TITLE
Does not work with chats from the new Edge's sidebar.

### ISSUE THREAD

**Issue by parabolee** [benf2004/Bing-Chat-History#11]
Anyway that feature can be added? Thanks, this tool is invaluable!

**Comment by benf2004** [comment]
I’ve tried doing this with little success. I’ll let you know if I figure it out.

**Comment by parabolee** [comment]
Great, thanks!

**Comment by ajhawkings** [comment]
If it helps, the sidebar uses this url https://edgeservices.bing.com/edgesvc/chat

**Comment by Butanium** [comment]
Any news on this ? I no longer use the browser conversation and use the sidebar most of the time :(

**Comment by benf2004** [comment]
I don't think this is possible based on my tests. Sorry. I'll keep you posted, but I think Edge blocks it from working.
